demos/atk-init.php

```diff
@@ -121,28 +124,24 @@
     $layout->addMenuItem(['Toast'], [$path . 'toast'], $adv);
     $layout->addMenuItem('Paginator', [$path . 'paginator'], $adv);
 
-    $path = '/demos/javascript/';
+    $path = $demosUrl . 'javascript/';
     $js = $layout->addMenuGroup(['Javascript', 'icon' => 'code']);
     $layout->addMenuItem('Events', [$path . 'js'], $js);
     $layout->addMenuItem('Element Reloading', [$path . 'reloading'], $js);
     $layout->addMenuItem('Vue Integration', [$path . 'vue-component'], $js);
 
-    $path = '/demos/others/';
+    $path = $demosUrl . 'others/';
     $other = $layout->addMenuGroup(['Others', 'icon' => 'plus']);
     $layout->addMenuItem('Sticky GET', [$path . 'sticky'], $other);
     $layout->addMenuItem('More Sticky', [$path . 'sticky2'], $other);
     $layout->addMenuItem('Recursive Views', [$path . 'recursive'], $other);
 
-//    $f = basename($_SERVER['PHP_SELF']);
-
-    $url = 'https://github.com/atk4/ui/blob/develop';
-
+    // Send to demos source page in github.
     $ex = [
-        new \atk4\ui\jsExpression('const baseUrl = []', [$url]),
+        new \atk4\ui\jsExpression('const baseUrl = []', ['https://github.com/atk4/ui/blob/develop']),
         new \atk4\ui\jsExpression('window.open(baseUrl + document.location.pathname, "_blank")'),
     ];
 
-    // Send to demos source page in github.
     \atk4\ui\Button::addTo($layout->menu->addItem()->addClass('aligned right'), ['View Source', 'teal', 'icon' => 'github'])
         ->on('click', $ex);
 
```

Comment:
@ibelar remove the `$img` below if it is not needed

Reply:
fine. not sure why it was there in first place.